.gitignore

```diff
@@ -22,6 +22,10 @@ vendor/
 *.js
 *.d.ts
 
+# Dart PB Files
```

Comment:
We should include the generated files too... no need to have them in source control.

Reply:
okay, wasn't certain on that with this change. will update